project3/style.css

```diff
@@ -0,0 +1,2586 @@
+}
+
+.identity {
+    padding-top: 52px;
+    width: 40%;
+    padding-bottom: 20px;
+}
+
+.personal {
+    border-bottom: none;
+    padding-top: 60px;
+    margin-bottom: -20px;
+}
+
+.characteristics {
+    padding-bottom: 20px;
+}
+
+.pna {
+    padding-top: 32px;
+}
+
+input,
+textarea {
+    font-family: BenchNine;
+    font-size: 18px;
+    font-weight: 500;
+    background-color: var(--color-grey);
+    display: block;
+    width: 295px;
+    height: 73px;
+    border: 2px solid var(--color-grey);
+}
+
+input[type="text"] {
+    margin-right: 16px;
+    margin-top: 12px;
+    margin-bottom: 30px;
+}
+
+
+input[type="number"] {
+    width: 109px;
+    height: 68px;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 2px solid var(--color-grey);
+    background-color: var(--color-offblack);
+    height: 24px;
+    width: 24px;
+    display: inline-block;
+    margin: auto;
+    margin-right: 4px;
+    margin-left: 2px;
+}
+
+input[type="radio"] {
+    border-radius: 50%;
+}
+
+fieldset {
+    border: 0;
+    padding: 0;
+}
+
+legend {
+    padding-top: 40px;
+}
+
+select {
+    cursor: pointer;
+    width: 240px;
+    height: 73px;
+    font-family: BenchNine;
+    font-size: 18px;
+    font-weight: 500;
+    margin-left: 24px;
+    margin-right: 24px;
+}
+
+.selectbox {
+    margin-bottom: 100px;
+}
+
+select:hover {
+    border: 5px solid var(--color-green);
+}
+
+.abilities {
+    text-align: center;
+    padding-bottom: 40px;
+
+}
+
+.category {
+    display: block;
+    padding: 4px;
+}
+
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    border-color: var(--color-green);
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    background-color: var(--color-green);
+}
+
+input[type="text"]:focus,
+input[type="email"]:focus,
+input[type="number"]:focus {
+    background-color: var(--color-offwhite);
+    border-bottom: 5px solid var(--color-green);
+}
+
+input[type="radio"]:checked+label,
+input[type="checkbox"]:checked+label {
+    color: var(--color-green);
+    text-decoration: underline var(--color-green);
+}
+
+textarea[id="powers"] {
+    height: 200px;
+    margin-top: 12px;
+    margin-bottom: 30px;
+    border-color: 2px solid var(--color-grey);
+}
+
+textarea[id="describe"]:focus,
+textarea[id="powers"]:focus {
+    background-color: var(--color-offwhite);
+    border-bottom: 5px solid var(--color-green);
+}
+
+
+@media(min-width: 375px) {
+    :root {
+        /*  ==================
+          COLOR PALETTE
+          ==================*/
+
+        --color-red: #c61a3a;
+        --color-green: #71f79f;
+        --color-offblack: #1e2019;
+        --color-grey: #d1d5de;
+        --color-offwhite: #fafafa;
+        --color-blue: #163346;
+        /* Reminder that you can name your variables anything you want, they just have to start with two dashes (--) */
+        /* ㅇㅣ거 하면 이름대로 색 바꿀 수 있음 */
+    }
```

Comment:
1. It is unlikely you need to start your media queries with 375px screen width -- mobile-first styles should include these in the default CSS, then you add media queries as the screen size grows and things need to change.
1. Only rules that change need to be updated in your media queries. As currently written, it is not possible for me to easily understand what your media queries are achieving because all the styles are repeated.